src/applications/auth/containers/AuthApp.jsx

```diff
@@ -216,8 +217,9 @@ export class AuthApp extends React.Component {
             <p>
               Please try again, and this time, select “Accept” on the final page
               of the identity verification process. Or, if you don’t want to
-              verify your identity with ID.me, you can try signing in with your
-              premium DS Logon or premium My HealtheVet username and password.
+              verify your identity with Login.gov or ID.me, you can try signing
```

Comment:
Are we fine with showing Login.gov in situations like this even if the `login_gov` flag is disabled?